src/astro/sql/operators/agnostic_save_file.py

```diff
@@ -25,28 +21,21 @@ class SaveFile(BaseOperator):
     :param output_file_format: file formats, valid values csv/parquet. Default: 'csv'.
     """
 
-    template_fields = (
-        "input_data",
-        "output_file_path",
-        "output_conn_id",
-        "output_file_format",
-    )
+    template_fields = ("input_data", "output_file")
 
     def __init__(
         self,
         input_data: Union[Table, pd.DataFrame],
-        output_file_path: str = "",
+        output_file: File,
         output_conn_id: Optional[str] = None,
```

Comment:
`self.output_conn_id` is not used anywhere, can be removed now that we have `output_file`?